Neos.Flow/Classes/Configuration/ConfigurationManager.php

```diff
@@ -512,6 +513,10 @@ protected function replaceVariablesInPhpString(string $phpString): string
             $replacement .= ($constantDoesNotStartAsBeginning ? $matchGroup['startString'] . "' . " : '=> ');
 
             if (isset($matchGroup['prefix']) && $matchGroup['prefix'] === 'env') {
+                $allowedTypes = ['int', 'bool', 'float', 'string'];
```

Comment:
hold on we dont use this line :P

Reply:
you're right. My first version threw an exception for unknown types

Reply:
Haha, someone I mentally saw that exception while reviewing. At first reading I also wondered where this is used, but somehow then I thought I had seen the Exception

Reply:
yes me too :joy: